aconfig.go

```diff
@@ -123,6 +136,11 @@ func (l *Loader) StopOnFileError() *Loader {
 
 // Build to initialize flags for a given configuration.
 func (l *Loader) Build() *Loader {
+	_, ok := l.config.FileDecoders[".json"]
+	if !ok {
+		l.config.FileDecoders[".json"] = &jsonDecoder{}
```

Comment:
I think you can move that to L72 via default FileDecoders

Reply:
lgtm